File: c/1007.c

// https://www.urionlinejudge.com.br/judge/pt/problems/view/1007

#include <stdio.h>
#include <stdlib.h>

int main()
{
    int a, b, c, d, diferenca;

    scanf("%d %d %d %d", &a, &b, &c, &d);

    diferenca = ((a * b)-(c * d));

    printf("DIFERENCA = %d\n", diferenca);

    return 0;
}

File: haskell/1007.hs

-- https://www.urionlinejudge.com.br/judge/pt/problems/view/1007

main :: IO ()
main = do a <- readLn :: IO Int
          b <- readLn :: IO Int
          c <- readLn :: IO Int
          d <- readLn :: IO Int
          putStrLn $ "DIFERENCA = " ++ show (a * b - c * d)